components/Header/index.tsx

```diff
@@ -29,144 +30,118 @@ interface HeaderProps {
   clear?: () => Promise<void>
 }
 
-interface State {
-  chosenRoute: string
-}
-
-class Header extends React.Component<HeaderProps, State> {
-  state: State = {
-    chosenRoute: '/',
-  }
-
-  connectAccount = async () => {
+const Header: React.FC<Props> = (props: Props) => {
```

Comment:
Note: I rewrote this to a function component initially for the toast implementation in tinlake-ui. This feature was implemented as a branch off that one. I've now removed all the toast functionality (since it's in Axis), but I didn't think it was necessary to undo the rewrite of this file to a function component.